[15.0][IMP] account_payment_order: Add a custom message if the invoice does not have a payment method.

victoralmau opened the pull request:
Add a custom message if the invoice does not have a payment method.

The message will be displayed when trying to create a line of a payment order to better clarify the reason.

Please @pedrobaeza can you review it?

@Tecnativa TT44551

pedrobaeza reviewed (commented):
There's already a check for no payment mode, but the code to get there is not correct and it should be done first:

https://github.com/OCA/bank-payment/pull/1118/files#diff-bb6441f4ea06462fe1aae1f093f82c4e57460864b29b209676a6c7ce14e49ef9L81

victoralmau committed:
[IMP] account_payment_order: Add a custom message if the invoice does not have a payment method

The message will be displayed when trying to create a line of a payment order to better clarify the reason.

TT44551

victoralmau commented:
Changes done.

pedrobaeza commented:
/ocabot merge patch
